x-pack/plugins/apm/public/components/app/Settings/anomaly_detection/index.tsx

```diff
@@ -77,7 +77,7 @@ export function AnomalyDetection() {
       <EuiText>
         {i18n.translate('xpack.apm.settings.anomalyDetection.descriptionText', {
           defaultMessage:
-            'The Machine Learning anomaly detection integration enables application health status indicators for each configured environment in the Service map by identifying transaction duration anomalies.',
+            `Machine Learning's anomaly detection integration enables application health status indicators for services in each configured environment by identifying transaction duration anomalies.`,
```

Comment:
CI is failing due to a formatting issue here. 

```
/dev/shm/workspace/parallel/10/kibana/x-pack/plugins/apm/public/components/app/Settings/anomaly_detection/index.tsx
  79:26  error  Delete `⏎···········`  prettier/prettier

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
```

Can you run prettier locally? 
```
yarn prettier  "./x-pack/plugins/apm/**/*.{tsx,ts,js}" --write
```